cql3/util.cc

```diff
@@ -19,9 +19,20 @@
 
 #include "util.hh"
 
+#ifdef DEBUG
+
+#include <ucontext.h>
+
+extern "C" {
+void __sanitizer_start_switch_fiber(void** fake_stack_save, const void* stack_bottom, size_t stack_size);
+void __sanitizer_finish_switch_fiber(void* fake_stack_save, const void** stack_bottom_old, size_t* stack_size_old);
+}
+
+#endif
+
 namespace cql3::util {
 
-void do_with_parser_impl(const sstring_view& cql, noncopyable_function<void (cql3_parser::CqlParser& parser)> f) {
+static void do_with_parser_impl_impl(const sstring_view& cql, noncopyable_function<void (cql3_parser::CqlParser& parser)> f) {
```

Comment:
A new rule of programming: any problem can be solved by adding another layer of "impl" :-)